Description: Avoid printing an error message when the settings file isn't found
Author: Stephen Kitt <steve@sk2.org>
Last-Update: 2011-03-13

--- gdmap.orig/src/preferences.c
+++ gdmap/src/preferences.c
@@ -658,7 +658,10 @@
 
     if (!g_key_file_load_from_file(file, filename, 0, &error))
     {
-        g_message(error->message);
+        if (error->code != G_FILE_ERROR_NOENT) {
+	    // The settings file is allowed to not exist
+            g_message(error->message);
+        }
         g_error_free(error);
         return;
     }
